### PR TITLE
[Rust] Bumped the version of wasmedge-sdk to v0.3.0

### PR DESCRIPTION
**Pull request by apepkuss**
In this PR, the version of `wasmedge-sdk` bumped from v0.2.0 to v0.3.0.

### CODE REVIEW

**Review by hydai — DISMISSED** [review]

**Comment by apepkuss** [comment]
@hydai The clippy issues have been fixed. In addition, the version of `wasmedge-sdk` bumps to 0.3.0, and the relevant docs are updated simultaneously. Please help review the latest changes, thanks a lot!

File: docs/book/en/src/embed/rust.md [hunk 1]
```diff
@@ -130,16 +131,20 @@ wasmedge-sdk = "0.1"
 
 ## `wasmedge-sys` crate
 
-`wasmedge-sys` serves as a wraper layer of `WasmEdge` C-API, and provides a group of safe low-level Rust interfaces. For those who are interested in using `wasmedge-sys` in their projects, you should also deploy the `WasmEdge` library on your local system as described in the [wasmedge-sdk crate](#wasmedge-sdk-crate) section. Then, copy/paste the following code in the `Cargo.toml` file of your project.
+`wasmedge-sys` serves as a wraper layer of `WasmEdge` C-API, and provides a group of safe low-level Rust interfaces. For those who are interested in using `wasmedge-sys` in their projects, you should also deploy the `WasmEdge` library on your local system as described in the [wasmedge-sdk crate](#wasmedge-sdk-crate) section. Then, copy/paste the following code in the `Cargo.toml` file of your project. For details, please reference [README](https://github.com/WasmEdge/WasmEdge/blob/master/bindings/rust/wasmedge-sys/README.md).
```
Comment (hydai) [inline docs/book/en/src/embed/rust.md:134]: `please refer to [README]`

**Review by hydai — CHANGES_REQUESTED** [review]

File: docs/book/en/src/embed/rust.md (hunk 1 above)
Comment (apepkuss) [reply docs/book/en/src/embed/rust.md:134]: Accepted. I'll fix the stupid mistake. Thanks a lot!

**Review by hydai — APPROVED** [review]
LGTM. Will merge after CI passes

**Comment by apepkuss** [comment]
@hydai Thanks for the review!

**Comment by apepkuss** [comment]
Hi @hydai,

The `binding-rust/windows` workflow failed here, but I tried it again on my Github Actions, and the workflow passed on all four platforms. I attached the snapshot here as a reference.

image

**Comment by apepkuss** [comment]
@hydai Thanks for the review!

**Pull request merged**